Update find-help-open-support-ticket.md (#119429)
Azure/azure-content: articles/active-directory-b2c/find-help-open-support-ticket.md

1. Sign in to the [Azure portal](https://portal.azure.com).

> 1. If you have access to multiple tenants, select the **Settings** icon in the top menu to switch to your Entra Workforce tenant from the **Directories + subscriptions** menu.  Currently, technical support cases cannot be created directly from an Azure Active Directory Business to Consumer (B2C) tenant.

Reviewer: ```suggestion
1. If you have access to multiple tenants, select the **Settings** icon in the top menu to switch to your Microsoft Entra tenant from the **Directories + subscriptions** menu. Currently, you can't submit support cases directly from your Azure AD B2C tenant.
```

Reviewer: @jasonfritts Please check this suggestion, then commit.

Author: #sign-off

Reviewer: @jasonfritts you need to commit the suggested changes, then I will sign off the PR. 

Thanks.
